src/handlebars/about.handlebars

```diff
@@ -7,53 +7,81 @@
+          AdoptOpenJDK.net started in 2017 following years of discussions about
+          the general lack of an open and reproducible build & test system for
+          <a target="_blank" href="https://openjdk.java.net">OpenJDK</a> and
+          <a target="_blank" href="https://www.eclipse.org/openj9/">Eclipse OpenJ9</a>
+          source across multiple platforms.
         </p>
 
         <p>
-          The intention is to provide binaries and hardware access for all of the major platforms, for the purposes of:
-          <ul>
-            <li>Allowing end users to have a reliable source of OpenJDK binaries for all platforms.</li>
-            <li>Research and Development by OpenJDK developers, academics and researchers.</li>
-            <li>An open, common, audited, build infrastructure for vendors to use (if they wish).</li>
-            <li>Providing a place to try out build infrastructure ideas that might one day be re-implemented in the <a href="https://openjdk.java.net">OpenJDK</a> project.</li>
-          </ul>
+          AdoptOpenJDK provides rock-solid pre-built OpenJDK binaries for the
+          Java ecosystem and also provides <a target="_blank" href="https://www.github.com/AdoptOpenJDK">infrastructure as code</a>,
+          and a <a href="https://ci.adoptopenjdk.net">Build farm</a> for builders of OpenJDK, on any platform.
+        </p>
+      </div>
+
+      <div class="margin-bottom">
+        <div class="callout">
+        <h2 class="bold">AdoptOpenJDK Mission</h2>
+
+          <ol>
+            <li>Provide a reliable source of OpenJDK binaries for all platforms, for the long term future!</li>
+            <li>Enable R&amp;D by OpenJDK developers, academics and researchers.</li>
+            <li>Provide an open, common, audited, build infrastructure for vendors to use (if they wish).</li>
+            <li>Provide a place to try out build and test infrastructure ideas that could be co-opted / re-implemented in <a target="_blank" href="https://openjdk.java.net">OpenJDK</a> and <a target="_blank" href="https://www.eclipse.org/openj9/">Eclipse OpenJ9</a>.</li>
+          </ol>
         </p>
+        </div>
       </div>
 
-      <div>
-        <h2 class="bold">About the AdoptOpenJDK Community</h2>
+      <div class="margin-bottom">
+        <h2 class="bold">Who's behind the AdoptOpenJDK.net Community?</h2>
         <p>
-          <a href="https://community.oracle.com/community/java/jcp/adopt-openjdk">AdoptOpenJDK</a> is a community of Java user group members, Java developers and vendors who are
-          advocates of <a href="https://openjdk.java.net">OpenJDK</a>, the open source project which forms the basis of the Java programming language and
-          platform.
+          <a href="https://community.oracle.com/community/java/jcp/adopt-openjdk">AdoptOpenJDK</a> is a
+          community of Java User Group (JUG) members, Java developers and vendors (including GoDaddy, IBM, jClarity, Microsoft, Pivotal and Red Hat)
```

Comment:
I understand the desire to use this vendor list, but it looks like an area we'll have to keep up to date.  Consider just making it a link to the sponsor page.